source/main/physics/Actor.cpp

```diff
@@ -4523,30 +4523,26 @@ bool Actor::getBrakeLightVisible()
 
 bool Actor::getCustomLightVisible(int number)
 {
-    if (number < 0 || number > 3)
+    if (number < 0 || number > 9)
     {
-        LOG("AngelScript: Invalid Light ID (" + TOSTRING(number) + "), allowed range is (0 - 4)");
+        LOG("AngelScript: Invalid Light ID (" + TOSTRING(number) + "), allowed range is (0 - 9)");
```

Comment:
remake pls: `fmt::format("invalid custom-light ID {}, allowed range is 0-{}", number, MAX_CLIGHTS-1)`